### PR TITLE
Feat(ui): Implement Telemedicine Page

### PR DESCRIPTION
**Pull request by hanihusam**
Partial work for #328 

## Description

Implementing `Telemedicine` page using Typesense backend.

## Current Tasks

- [x] Telemedicine CTA on Homepage
- [x] Update Telemedicine CTA on homepage to link to new Telemedicine page
- [x] Create the static JS dummy data
- [x] Test

### CODE REVIEW

**Comment by mazipan** [comment]
Split the PR.
Please focus to ship it first.
Use the static data for first attempt.
Add the improvement on the next iteration.

**Comment by hanihusam** [comment]
Should I use this [link](https://docs.google.com/spreadsheets/d/1gGnIiXmCcGpMkB_h98FiHW99uVdwcdTBq_GhIEl_4eM/edit?usp=sharing) for telemedicine CTA for this PR or direct to the new Telemedicine page with static dummy data? @mazipan @zainfathoni

**Comment by mazipan** [comment]
I suggest to create a dummy page, using a static JS data

**Comment by resir014** [comment]
For the CTA on the homepage, we can simply use an image for it, just like in the Sembako CTA.

I'll export it from the Figma soon.

**Comment by mazipan** [comment]
How about the href link @resir014 ?
Is it okay to redirect it to the Google Sheet for now?

**Comment by hanihusam** [comment]
Okay, Mas. I am waiting for your decision. Either direct to Google Sheet or to telemedicine page, I am still waiting for the image file.

**Comment by zainfathoni** [comment]
I don't think so. This ticket is supposed to include the telemedicine page implentation as well, even though it's stil hardcoded.

**Comment by resir014** [comment]
Agree with Mas @zainfathoni. Let's hardcode some values first, the backend can follow on a separate PR. I'll edit the first post's description so it won't automatically close the issue in question.

Also @hanihusam, here's the Cloudinary URL of the Telemedicine CTA: `v1628431903/telemedicine-banner_2x_gonhgo.png`

**Comment by hanihusam** [comment]
What components/files on this issue should I test? Anyway, I use Kontak Darurat data for the temporary data of telemedicine page.

**Comment by zainfathoni** [comment]
> What components/files on this issue should I test? Anyway, I use Kontak Darurat data for the temporary data of telemedicine page.

Please test the files that you have changed.
You can learn how to check the code coverage of each files by reading [this blog post](https://www.emgoto.com/jest-code-coverage/).

**Comment by mazipan** [comment]
The dummy data should be based on this https://docs.google.com/spreadsheets/d/1gGnIiXmCcGpMkB_h98FiHW99uVdwcdTBq_GhIEl_4eM/edit#gid=0

cc @zainfathoni

**Comment by mazipan** [comment]
You can use this @hanihusam

https://gist.github.com/mazipan/15e29316ba7a24d0066d027096240c88

**Comment by mazipan** [comment]
https://www.flaticon.com/free-icon/doctor_387562?term=doctor&page=1&position=32&page=1&position=32&related_id=387562&origin=tag

This icon can be an avatar placeholder maybe 😂

File: components/telemedicine/telemedicine-contact-card.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          </div>
+        </div>
+      </div>
+      <div className="p-3 justify-center">
+        <PrimaryAnchorButton
+          block
+          className="bg-light-blue-400 font-normal focus:bg-light-blue hover:bg-light-blue-600 text-white"
+          color="none"
+          data-testid={`contact-button-${contact.doctor_name}`}
+          href={contact.contact}
```
Comment (mazipan) [inline components/telemedicine/telemedicine-contact-card.tsx:46]: the URL should be:


```js
const getLink = (platform, phone) => {
  if (platform.toLowerCase() === 'whatsapp') {
   // WA URL can not use 08, should use 628
    return 'https://wa.me/' + phone.replace(new RegExp(\^08\), 628)
  } else if (platform.toLowerCase() === 'telegram') {
    return 'https://t.me/' + phone
  } else {
    return '' // not available for Line
  }
}
```

Comment (hanihusam) [reply components/telemedicine/telemedicine-contact-card.tsx:46]: Thanks Mas for the code, haven't had time to do it yet 😅

**Comment by hanihusam** [comment]
I forgot `homepage-telemedicine-cta` test. I will work on it.

**Review by hanihusam — COMMENTED** [review]
Can you help me with this Mas @zainfathoni?

**Comment by zainfathoni** [comment]
> Can you help me with this Mas @zainfathoni? In the `get-contact-links` test file.

Sure, I'll take a look at it later today.

**Comment by mazipan** [comment]
This PR block #628 

Can we speed up? or #628 will merged first, and it cause your changes need to be reworked.

**Comment by mazipan** [comment]
Help to align this PR with our latest changes on Layout ya @hanihusam.

**Comment by mazipan** [comment]
Can we just show grey placeholder on Avatar part @hanihusam ?
It's weird to see a "man-name" doctor with a women avatar.

**Comment by hanihusam** [comment]
Ready to 🚢

**Review by mazipan — APPROVED** [review]
Looks good to me

**Pull request merged**